plugins/GSdx/Renderers/HW/GSRendererHW.cpp

```diff
@@ -812,8 +815,11 @@ void GSRendererHW::SwSpriteRender()
 				sc = GSVector4i::loadl(&s[scol[x]]).u8to16();  // 0x00AA00BB00GG00RR00aa00bb00gg00rr
 
 				// Apply TFX
-				ASSERT(m_context->TEX0.TFX == 0);
-				sc = sc.mul16l(vc).srl16(7).clamp8();  // clamp((sc * vc) >> 7, 0, 255), srl16 is ok because 16 bit values are unsigned
+				ASSERT(tex0_tfx == 0 || tex0_tfx == 1);
+				sc = tex0_tfx == 0 ?
+					sc.mul16l(vc).srl16(7).clamp8() : // clamp((sc * vc) >> 7, 0, 255), srl16 is ok because 16 bit values are unsigned
+					sc
```

Comment:
Would avoiding the ternary improve readability?
```
if (tex0_tfx == 0)
    sc = sc.mul16l(vc).srl16(7).clamp8() : // clamp((sc * vc) >> 7, 0, 255), srl16 is ok because 16 bit values are unsigned
```

Reply:
Sure